domains/transfer-agreements/TransferAgreements.API/API/ApiModels/Responses/TransferAgreementResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+
+namespace API.ApiModels.Responses
+{
+    public record TransferAgreementResponse(Guid Id, long StartDate, long EndDate, string ReceiverTin);
```

Comment:
this should be `TransferAgreementsResponse(List<TransferAgreement> transferAgreements)` or it should be deleted and just let the controller return `list<TransferAgreement>`

Reply:
I have made a wrapper class which represents a list of TransferAgreementResponse.